flask_restless/manager.py

```diff
@@ -64,8 +65,10 @@
 #: - `blueprint_name`, the name of the blueprint that contains this API,
 #: - `serializer`, the subclass of :class:`Serializer` provided for the
 #:   model exposed by this API.
+#: - `primary_key`, the primary key used by the model
 #:
-APIInfo = namedtuple('APIInfo', 'collection_name blueprint_name serializer')
+APIInfo = namedtuple(
+    'APIInfo', 'collection_name blueprint_name serializer primary_key')
```

Comment:
This is getting a little long, why don't we make the second argument to `namedtuple` a list of strings:

    APIInfo = namedtuple('APIInfo', ['collection_name', 'blueprint_name', 'serializer',
                                     'primary_key'])